src/impl/MilvusClientImpl.cpp

```diff
@@ -484,7 +484,39 @@ MilvusClientImpl::CreateIndex(const std::string& collection_name, const IndexDes
 Status
 MilvusClientImpl::DescribeIndex(const std::string& collection_name, const std::string& field_name,
                                 IndexDesc& index_desc) {
-    return Status::OK();
+    auto pre = [&collection_name, &field_name]() {
+        proto::milvus::DescribeIndexRequest rpc_request;
+        rpc_request.set_collection_name(collection_name);
+        rpc_request.set_field_name(field_name);
+        return rpc_request;
+    };
+
+    auto post = [&index_desc](const proto::milvus::DescribeIndexResponse& response) {
+        auto count = response.index_descriptions_size();
+        if (count > 0) {
```

Comment:
no need to check `if (count > 0)`, the for statement already has such logic test.